components/mediation/data-publishers/org.wso2.micro.integrator.observability/pom.xml

```diff
@@ -66,8 +75,9 @@
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Axis2Module>${project.artifactId}-${project.version}</Axis2Module>
                         <Export-Package>
-                            org.wso2.micro.integrator.prometheus.observability.* ;version="${project.version}"
+                            org.wso2.micro.integrator.observability.* ;version="${project.version}"
                         </Export-Package>
+                        <Private-Package>io.prometheus.client.*</Private-Package>
```

Comment:
Why are we private packaging here?

Reply:
This was done to make the io.prometheus.client.* package to copy into the bundle but not export.
This was done to fix the issue com.sun package not found thrown during the build phase.